packages/desktop-client/src/components/reports/reports/SpendingCard.tsx

```diff
@@ -60,10 +64,17 @@ export function SpendingCard() {
                   ...styles.mediumText,
                   fontWeight: 500,
                   marginBottom: 5,
+                  color: !difference
+                    ? 'inherit'
+                    : difference <= 0
+                      ? theme.noticeTextLight
```

Comment:
@carkom aren't these colours reversed?  It's currently showing red when this month's difference is a positive

Reply:
Isnt that supposed to mean "you spent more than last month try not to do that again" so a positive difference is bad and negative difference is good?

Reply:
So positive is indicating you've overspent compared to last month.

Reply:
oooh, of course :facepalm: